deployment/updates/scripts/xml/notifications/entryClipJobStartedHttpNotificationTemplate.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@TalDubovKaltura FYI for next time the file name format should be 
entryClipJobStartedHttpNotification.template.xml and not
entryClipJobStartedHttpNotificationTemplate.xml

Reply:
@yossipapi Right, so next time I will add a pull request I will change this too